app/services/katello/event_daemon/monitor.rb

```diff
@@ -0,0 +1,52 @@
+module Katello
```

Comment:
I vote to move runner and monitor to app/lib vs app/services (I did this too as part of my katello-agent pr) reason being that they are not "service classes" in the sense that they encapsulate logic is used by other areas of the app. Really this is part of an underlying subsystem of the app - therefore lib seems more appropriate! The distinction is a bit vague...